doc/development/plugins.rst

```diff
@@ -472,59 +349,10 @@ where
 
 * ``devices_list`` is a list of devices you would like to register,
 
-* ``example.mydevice1`` is the short name of the device, and
+* ``example.mydevice1`` is the name of the device, and
 
 * ``MyModule.MySubModule`` is the path to your Device class, ``MyDevice1``.
 
 To ensure your device is working as expected, you can install it in developer mode using
 ``pip install -e pluginpath``, where ``pluginpath`` is the location of the plugin. It will
 then be accessible via PennyLane.
-
-Testing
--------
-
-All plugins should come with extensive unit tests, to ensure that each logical unit of the
-device has correct execution.
-
-Integration tests to check that the probabilities, expectation values, variance, and samples are
-correct for various circuits and observables are provided as part of the PennyLane device
-test utility:
-
-
-.. code-block:: console
-
-    pl-device-test --device device_shortname --shots 10000
-
```

Comment:
@albi3ro do the device integration tests not work with the new device API?

Reply:
They do. This page is just a work in progress.

Reply:
thanks!